package.json

```diff
@@ -36,7 +36,7 @@
     "@edx/frontend-component-site-header": "2.3.0",
     "@edx/frontend-i18n": "2.1.0",
     "@edx/frontend-logging": "2.1.0",
-    "@edx/paragon": "^6.2.0",
```

Comment:
I'm not sure how this change (from Renovate) ended up in here.  Sorta concerning.